spec/classes/config_spec.rb

```diff
@@ -7,214 +7,222 @@
         facts
       end
 
-      it { should compile }
+      it { is_expected.to compile }
 
       context 'secrets' do
         context 'w/o secret' do
-          let(:params) { {
-              :options => {}
-          } }
+          let(:params) { { :options => {} } }
```

Comment:
I usually leave out the spaces inside the braces for hash literals and keep them for blocks, just to make them a little easier to visually differentiate. This would then become `let(:params) { {:options => {}} }`.